### ISSUE TITLE
Missing definitions for latest Parse Platform

### ISSUE THREAD

**Issue by REPTILEHAUS** [DefinitelyTyped/DefinitelyTyped#35996]
I tried using the `@types/parse` package and had problems.

The following are missing definitions

_registerAuthenticationProvider, _linkWith, _isLinked, _unlinkFrom

```
Parse.User._registerAuthenticationProvider(provider);
await user._linkWith(provider.getAuthType(), provider.getAuthData());
user._isLinked(provider); // true
user._unlinkFrom(provider.getAuthType());   
```

@dpoetzsch 
@jaeggerr
@flavionegrao 
@wesleygrimes 
@agoldis 
@AlexandreHetu 
@dplewis 
@yomybaby 
@jlnquere 
@yagotome

**Comment by agoldis** [comment]
@REPTILEHAUS thanks for discovering those missing definitions! 

The definitions are collaboratevly supported by community of people just like you, I would encourage you to use this opportunity and create a PR that adds the missing definitions, by yourself :) 

Take a look at already merged PR as an example: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35894

**Comment by REPTILEHAUS** [comment]
Sure will do it tomorrow, I ended up fixing it with some loose types

**Comment by kentrh** [comment]
@agoldis This can be closed ref #42651

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
